neurips23/ongoing_leaderboard/leaderboard.md

```diff
@@ -27,8 +27,6 @@ Note: "pinecone_smips" and "zilliz" are not open source
 ### Streaming track
 The [result file](streaming/res_final_runbook_AzureD8lds_v5.csv) lists measurements for all streaming algorithms on Azure D8lds_v5. 
 
-While the highest achieved recall@10 is 0.99786 (by pinecone), the similar submission by zilliz has almost the same recall at 0.99768. Since in the current setup of the track it seems that there is little innovation left, this streaming track in its current form will no longer be tracked in future leaderboards.
-

```

Comment:
I wasn't part of the discussion about this decision. @ingberam Do you agree of bringing this track back?

Reply:
see comment on the main thread. No objection in principle, the reason was that the results were saturating at 0.99+ recall. 
The thing is, that the current PR shows different results for the *existing* algorithms, which is very strange.